samcli/commands/check/resources/Warning.py

```diff
@@ -0,0 +1,12 @@
+"""
+Class object for warnings. Warning messages, as well as the warning type is stored here
+"""
+
+
+class CheckWarning:
+    warning_type: str
+    message: str
+
+    def __init__(self):
```

Comment:
Does it make sense to create a CheckWarning object without warning_type or message?
If not, the constructor shouldn't be empty and we should set the class members with parameters taken in from the constructor.

Reply:
`warning_type` was removed, as it is no longer needed. A message must now be passed when `CheckWarning` object is created